config/default_records/submission_templates/005_bioscan_submission_templates.yml

```diff
@@ -7,11 +7,13 @@ Limber-Htp - Bioscan Lysate Prep:
     request_type_keys: ["limber_bioscan_lysate_prep"]
     product_line_name: Bioscan
     product_catalogue_name: Bioscan
-# Library preparation from the Lysate plate
+# Library preparation from the Lysate Input plate
 # NB. PacBio long read Sequencing happens in Traction for Bioscan
-Limber-Htp - Bioscan Library Prep:
```

Comment:
The old template needs superceded_by (note the spelling) to hide it from use, either (SUPERCEDED_BY_UNKNOWN_TEMPLATE = -2) or the id of the new submission. It could be added to deployment notes.

Reply:
Yes ok, will add to deployment notes